pvlib/atmosphere.py

```diff
@@ -337,6 +337,86 @@ def gueymard94_pw(temp_air, relative_humidity):
     return pw
 
 
+def rh_from_tdew(temperature, dewpoint, coeff=(6.112, 17.62, 243.12)):
```

Comment:
I noticed earlier but forgot to comment: we should rename this parameter to `temp_dew`, the name used by other pvlib models and iotools functions.  I'll push a commit for that to give @kurt-rhee a break from this PR :P